src/univariate/continuous/normal.jl

```diff
@@ -1,12 +1,18 @@
-immutable Normal <: ContinuousUnivariateDistribution
-    μ::Float64
-    σ::Float64
+immutable Normal{T<:Number} <: ContinuousUnivariateDistribution
+    μ::T
+    σ::T
 
-    Normal(μ::Real, σ::Real) = (@check_args(Normal, σ > zero(σ)); new(μ, σ))
-    Normal(μ::Real) = Normal(μ, 1.0)
-    Normal() = Normal(0.0, 1.0)
+    Normal(μ::T, σ::T) = new(μ, σ)
```

Comment:
You are right, I don't want to remove it... I just wasn't sure if it is correct syntactically to retain more than one inner constructor in a situation like this (the `Rational` example in the official Julia doc for constructors defines exactly one inner constructor for `Rational` and sets the rest of them as outer constructors...